.pre-commit-config.yaml

```diff
@@ -52,6 +52,12 @@ repos:
         args: [--py36-plus]
         name: Upgrade code
 
+  - repo: https://github.com/myint/docformatter
+    rev: v1.4
+    hooks:
+      - id: docformatter
+        args: [--in-place, --wrap-summaries=115, --wrap-descriptions=120]
```

Comment:
how to trigger this check? it does not seem to run automatically. when it is blocking prs, people probably don't know what's up

Reply:
it only says: 

```
docformatter.............................................................Failed
- hook id: docformatter
- files were modified by this hook
```

Reply:
sometimes you need to run it multiple times...
but the call for individual formatting is `docformatter . -ri --wrap-summaries=115 --wrap-descriptions=120`

Reply:
@awaelchli @Borda 

Ok, I've fixed that. Indeed it was a problem since I've been using all pre-commit hook commands to fix only files updated in this branch, but there was a problem with `./tests/trainer/loops/test_training_loop.py` as well.

You can  use `-c, --check` command in your hook, so wrong files will be displayed.

Reply:
will be only displayed or also fixed?

Reply:
@Borda 
Ti will only display wrongly formatted files:

> -c, --check           only check and report incorrectly formatted files

Why it should fix the problem?
All other commit hooks didn't do that, only displayed problems I had to fix them on my own.

Reply:
that is not true, most of the others does fixing, see for example Isort, black, pyugrade, ...
in fact only one reporting issues is flake8 as it does not offer any fixing at all :]